lib/components/app/desktop-nav.js

```diff
@@ -1,29 +1,40 @@
+import PropTypes from 'prop-types'
 import React from 'react'
 import { Nav, Navbar } from 'react-bootstrap'
 import { connect } from 'react-redux'
 
 import NavLoginButtonAuth0 from '../user/nav-login-button-auth0.js'
 import { accountLinks, getAuth0Config } from '../../util/auth'
-
 import AppMenu from './app-menu'
 
-const AppNav = ({ otpConfig }) => {
+/**
+ * The desktop navigation bar, featuring a sign-in button/menu.
+ * TODO: merge with the mobile navigation bar.
+ */
+const DesktopNav = ({ otpConfig, title }) => {
```

Comment:
I'm still a little confused as to why there is a `title` prop. Shouldn't this be taking the title from `otpConfig`?

Reply:
I don't know! `OpenTripPlanner` was hardcoded as title before, and `trimet-mod-otp` implementations don't use a title at all, so I was trying to accommodate both.

Reply:
OK, so previously `example.js` just had a very barebones navbar and was just serving as an example for how to piece together otp-react-redux components within a web frame. However, now that we have otp-ui, I think it's OK for `example.js` to change from that original purpose somewhat. That's all just backstory to give you a little more context. For this component: just remove the title prop, get the title from otpConfig, and set the default value to OpenTripPlanner. There's no reason to have a `title` prop, when it's not being used by anything -- that's just dead code that someone down the road will think is important for some unknown reason.

Reply:
Using title from otpConfig in bb9f9e8926a9e638be4f1a9a33340e6c48792067.